dnf+

```diff
@@ -0,0 +1,19 @@
+---
+tags: [ packaging ]
+---
+# To install a package:
+dnf install <package>
+
+# To find packages matching <phrase>:
+dnf search <phrase>
+
+# To find which package provides an executable:
+dnf provides <executable>
+
+# The following are available after installing "dnf-plugins-core"
+
+# To download a package:
+dnf download <package>
```

Comment:
Multiple fixes available: please chose one of them.

```suggestion
# Download a specific package:
dnf download "<package>"
```

Almost the same as above.

```suggestion
# Download specific packages:
dnf download "<package1 package2 ...>"
```

- Almost the same as above.
- Optional change: if `dnf` can accept multiple packages at once for `download` command then u can use this fix.